### ISSUE TITLE
pac paportal upload doesn't delete adx_entitypermission_webrole records 

### ISSUE THREAD

**Issue by ThierryBIENVENU** [microsoft/powerplatform-vscode#327]
If we remove relation between portal WebRole and Entity Permission on dev environment:
- After the 'pac paportal download', Yaml of Entity Permission is correctly updated on file system (webrole id is removed from entity permission webrole list) 
- but the pac paportal upload doesn't remove the relationship in other environment. The relationship is an N:N relation and must be deleted with dissociate dataverse message.

I thinks this relationship should be part of the manifest.yml

**Comment by MrRishabhJain** [comment]
Agreed. I tried this and was able to reproduce this bug on my end. Are you able to work around this bug by deleting the unwanted permission records from the target org manually after the upload?

**Comment by MrRishabhJain** [comment]
Upon further investigation, it was found that our current implementation does not support migration of these relationships via PAC. We will take this up as an enhancement based on priority.

**Comment by ThierryBIENVENU** [comment]
I see your responses today,

> Agreed. I tried this and was able to reproduce this bug on my end. Are you able to work around this bug by deleting the unwanted permission records from the target org manually after the upload?

Doing something manually is not a perfect way to automate the deployment :) and it is not the best entity to use a manual step, and possibly manual errors, as it affects the security of the portal.
As a workaround, I developed a power automate to remove all relationships before uploading the portal changes, but this is not the best solution because if something goes wrong during deployment, the portal data may remain unavailable.

> Upon further investigation, it was found that our current implementation does not support migration of these relationships via PAC. We will take this up as an enhancement based on priority.

"current implementation does not support migration", in my opinion, the current implementation should support the migration of all portal entities, if we can't automate the whole process, it's a bug by my definition.
Anyway, why did you close the issue before its implementation?

**Comment by MrRishabhJain** [comment]
@ThierryBIENVENU, we have added the requested functionality and now any deletions made to entity relations should reflect in Dataverse as well upon 'pac paportal upload'.

This will be available in the upcoming January update. Hope this helps.
